tests/unit/core/test_deploy_export.py

```diff
@@ -349,4 +464,47 @@ def test_hf_bert(self, tmpdir, df_type):
         # Generate filename in the temporary directory.
         tmp_file_name = str(tmpdir.mkdir("export").join("bert"))
         # Test export.
-        self.__test_export_route(module=bert, out_name=tmp_file_name, mode=df_type, input_example=input_example)
+        self.__test_export_route(
+            module=bert, out_name=tmp_file_name, mode=df_type, input_example=input_example, onnx_opset=11
+        )
+
+    @pytest.mark.unit
+    @pytest.mark.run_only_on('GPU')
+    @pytest.mark.parametrize(
+        "df_type", [DF.ONNX, DF.TORCHSCRIPT, DF.PYTORCH]  # , pytest.param(DF.TRTONNX, marks=requires_trt)]
+    )
+    def test_waveglow(self, tmpdir, df_type):
+        url = "https://api.ngc.nvidia.com/v2/models/nvidia/waveglow_ljs_256channels/versions/2/files/waveglow_256channels_ljs_v2.pt"
+        ptfile = "./waveglow_256channels_ljs_v2.pt"
```

Comment:
This isn't the NeMo checkpoint. Why are you testing this?

Reply:
wget https://api.ngc.nvidia.com/v2/models/nvidia/waveglow_ljspeech/versions/2/files/WaveGlowNM.pt